### PR TITLE
Removed desktop path from the library section

### PR DESCRIPTION
**Pull request by yair100**
For #3869

### CODE REVIEW

File: Files/Filesystem/LibraryManager.cs [hunk 1]
```diff
@@ -110,7 +110,6 @@ private async Task SyncLibrarySideBarItemsUI()
                             MainPage.SideBarItems.Insert(1, librarySection);
 
                             libraryItems.Clear();
-                            libraryItems.Add(AppSettings.DesktopPath);
```
Comment (d2dyno1) [inline Files/Filesystem/LibraryManager.cs:113]: Why? It was useful to me!

Comment (yaichenbaum) [reply Files/Filesystem/LibraryManager.cs:113]: @d2dyno1 The desktop path isn't a default library, until we add proper support for libraries, I think we should try to follow the default behavior.

Comment (gave92) [reply Files/Filesystem/LibraryManager.cs:113]: Then should we move desktop folder to the first section with the recyclebin?

**Comment by XPoppyX** [comment]
"Downloads" is not part of the library either,.....I think

**Comment by tsvietOK** [comment]
I think both Downloads and Desktop are parts of library.
image

**Comment by XPoppyX** [comment]
Captura de pantalla 2021-03-15 185206

**Comment by tsvietOK** [comment]
@XPoppyX Agreed. Desktop and Downloads are still usable in Libraries section.

**Comment by lukeblevins** [comment]
@yair100 We currently don't fully support libraries in Files. While this can change, we need to rename the Libraries section to Locations so users don't get false expectations regarding the grouping functionality available in Explorer.

**Comment by d2dyno1** [comment]
> @yair100 We currently don't fully support libraries in Files. While this can change, we need to rename the Libraries section to Locations so users don't get false expectations regarding the grouping functionality available in Explorer.

How about naming it "Quick Access"?

File: Files/Filesystem/LibraryManager.cs (hunk 1 above)
Comment (yaichenbaum) [reply Files/Filesystem/LibraryManager.cs:113]: @gave92 The desktop location is already one of the default locations pinned to the favorite section.

**Comment by yair100** [comment]
> @yair100 We currently don't fully support libraries in Files. While this can change, we need to rename the Libraries section to Locations so users don't get false expectations regarding the grouping functionality available in Explorer.

@duke7553 @nvi9 is working on adding proper support for libraries so that shouldn't make a difference for long.

File: Files/Filesystem/LibraryManager.cs (hunk 1 above)
Comment (gave92) [reply Files/Filesystem/LibraryManager.cs:113]: Keep forgetting about that, for me it's a yes

**Review by gave92 — APPROVED** [review]

**Pull request merged**